canvas_exercise/shapes_game/index.js

```diff
@@ -26,11 +14,166 @@ window.addEventListener("load", function() {
       seconds = 3,
       intervalId;
 
-  canvas.width = width;
-  canvas.height = height;
+      canvas.width = width;
+      canvas.height = height;
+
+  var shapeVal = 0;
+  var totalScore = 0;
+  var timerCountDown = 1;
+  var keyCodes = [37, 38, 39, 40];
+
+
+//-------------------------------------------------------------------  
+//Opening screen
+drawStartText();
+
+//-------------------------------------------------------------------  
+//Start Game
+  function startGame() {
+    clear(); //clear the canvas
+    drawRandomShape(); //add a random shape track scoring 
+    countDown(); //start timer countdown
+    endGaame() //use as a callback for when timer countdown expires  
```

Comment:
Looks like this is a typo.  But I think that typo was actually making your code work.  You don't want to call endGame there because your game will never start.